### DISCUSSION TITLE
How to modify block

### DISCUSSION THREAD

**Discussion by tauleshwar** [WordPress/gutenberg#44047] [Q&A]
Hello friends, I wanted to know that Can we add new attributes while creating the variation of core blocks, 

I want to add language attributes in code blocks

**Comment by t-hamano** [comment]
Hi. 
You can define new attributes when you run `registerBlockVariation`. [This document](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-variations.md) will help.
